funding/manager.go

```diff
@@ -1012,6 +1080,22 @@ func (f *Manager) stateStep(channel *channeldb.OpenChannel,
 	// The channel was added to the Router's topology, but the channel
 	// announcement was not sent.
 	case addedToRouterGraph:
+		if channel.ChanType.IsZeroConf() {
```

Comment:
Only if it isn't an unadvertised channel? Though I may be missing that check elsewhere.

Reply:
even in the non-zero-conf case this is called for private channels, it's not announced though until `annAfterSixConfs` and only if public